### ISSUE TITLE
PointsGenerator ignores radiance_min and radiance_max

### ISSUE THREAD

**Issue by aturcotte** [EMVA1288/emva1288#174]
When you create a `PointsGenerator`, you can pass in `radiance_min` and `radiance_max`, as well as `exposure_fixed`. There are two different cases for how the initialization takes place:

1. If you don't pass `exposure_fixed`, `radiance_min` and `radiance_max` are ignored and we use `self._radiance = self._cam.get_radiance_for()`.
2. If you pass `exposure_fixed`, the camera's exposure is set accordingly, then we compute `self._radiance_min = self._cam.get_radiance_for(mean=target)` and `self._radiance_max = self._cam.get_radiance_for()`.

So basically, there's no way to externally control the radiance via `radiance_min` and `radiance_max`, because we either ignore or clobber them.
